examples/tools/ModelStructureExample.ipynb

```diff
@@ -0,0 +1,556 @@
+      "\u001b[0;32m~/GitHub/bokeh/bokeh/io/webdriver.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m    161\u001b[0m \u001b[0;31m#-----------------------------------------------------------------------------\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    162\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 163\u001b[0;31m \u001b[0mwebdriver_control\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_WebdriverState\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    164\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    165\u001b[0m \u001b[0matexit\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mregister\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;32mlambda\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mwebdriver_control\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;32m~/GitHub/bokeh/bokeh/io/webdriver.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, reuse, kind)\u001b[0m\n\u001b[1;32m    113\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkind\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m\"firefox\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    114\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 115\u001b[0;31m                 raise RuntimeError(\"Neither firefox/geckodriver nor chromium-browser/chromedriver are available on system PATH. \" \\\n\u001b[0m\u001b[1;32m    116\u001b[0m                                    \"You can install the former with 'conda install -c conda-forge firefox geckodriver'.\")\n\u001b[1;32m    117\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mRuntimeError\u001b[0m: Neither firefox/geckodriver nor chromium-browser/chromedriver are available on system PATH. You can install the former with 'conda install -c conda-forge firefox geckodriver'."
+     ]
+    }
+   ],
+>>>>>>> jeremy
```

Comment:
This and other instances need to be cleaned up, both merge markers and outputs. In general, commited notebooks must not have any outputs.

Reply:
I'm taking care of this.